.gitignore

```diff
@@ -1,2 +1,6 @@
 node_modules
-node_modules/*
\ No newline at end of file
+node_modules/*
+src/*.b
+yarn-error.log
+yarn.lock
+package-lock.json
```

Comment:
Excluding package-lock will lead to inconsistency in package version, and troubling debugging